tests/update/updates_packagekit_gpk.pm

```diff
@@ -71,8 +76,14 @@ sub run {
             # Wait until installation is done
             assert_screen \@updates_installed_tags, 3600;
             if (match_has_tag("updates_none")) {
-                send_key "ret";
-                last;
+                send_key 'ret';
+                if (check_screen "updates_installed-restart") {
```

Comment:
Why not use `match_has_tag` here? We don't want to waste time with check_screens which don't match by default. If you really have to, which I don't think is necessary, use `check_screen '<tag>', 0;`

Reply:
Because the screen changes when pressing return and no other assert or check screen is done meanwhile. Timeout will be added with next push